tests/unit/hive_metastore/test_mapping.py

```diff
@@ -211,6 +211,29 @@ def test_skip_happy_path(caplog):
     assert len(caplog.records) == 0
 
 
+def test_unskip(caplog):
+    ws = create_autospec(WorkspaceClient)
+    sbe = create_autospec(SqlBackend)
```

Comment:
You can use `MockBackend`, get it with the `mock_backend` fixture. And use the `mock_backend.queries` for your asserts below